mslice/plotting/plot_window/plot_options.py

```diff
@@ -312,24 +303,27 @@ def __init__(self, line_options, color_validator):
         row3.addWidget(self.marker_label)
         row3.addWidget(self.line_marker)
 
+        self.error_bar_checkbox = QtWidgets.QCheckBox("Show Line Error Bars")
```

Comment:
I think 'line' is redundant and this could just be 'Show Error Bars'

Reply:
Alright